hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/TokenServiceImpl.java

```diff
@@ -138,10 +139,36 @@ public void migrate(@NonNull MigrationContext ctx) {
                 addStakingAccounts(asAccount(801), accounts, FUNDING_ACCOUNT_EXPIRY);
 
                 updateNetworkRewards(ctx);
+                updateStakingNodeInfo(ctx);
             }
         };
     }
 
+    private void updateStakingNodeInfo(final MigrationContext ctx) {
+        // TODO: This need to go through address book and set all the nodes
```

Comment:
This is a pretty big TODO.  Do we have an issue for it?
> If we have an issue, perhaps use `@todo('####')` syntax so the issue number is right here.

Reply:
Don't have an issue yet, since it comes under what happens on migrate, which is Matt's issue. This will probably be done in or after Matt's PR is merged.